project-base/tests/ReadModelBundle/Functional/Twig/ImageExtensionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Functional\Twig;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Image\AdditionalImageData;
+use Shopsys\FrameworkBundle\Component\Image\ImageFacade;
+use Shopsys\FrameworkBundle\Twig\ImageExtension as FrameworkImageExtension;
+use Shopsys\ReadModelBundle\Image\ImageView;
+use Shopsys\ReadModelBundle\Twig\ImageExtension;
+use Tests\ShopBundle\Test\FunctionalTestCase;
+
+class ImageExtensionTest extends FunctionalTestCase
+{
+    public function testGetImageHtmlWithMockedImageFacade(): void
+    {
+        $frameworkBundleImageExtension = $this->getContainer()->get(FrameworkImageExtension::class);
+        $domain = $this->getContainer()->get(Domain::class);
+
+        $productId = 2;
```

Comment:
I suggest to inline these variables as I don't see a benefit in their reuse. The `$extension` variable is even confusing as I expected `ImageExtension` in it, not a string

Reply:
Test is now completely different, please check it out.

Reply:
ok, I'll check it